h/schemas/api/moderation.py

```diff
@@ -0,0 +1,50 @@
+import colander
+
+from h.models.annotation import ModerationStatus
+
+
+class ModerationStatusNode(colander.SchemaNode):
+    schema_type = colander.String
+    validator = colander.OneOf(["APPROVED", "PENDING", "DENIED", "SPAM"])
+
+    def deserialize(self, cstruct):
+        return ModerationStatus(cstruct) if cstruct else None
```

Comment:
This seems to be right method to override to alter "appdata". Mutating cstruct in place seems to work on Schemas (because it's a dict) but on a node cstruct is just a string in this case.

Reply:
So this is just because we want validation to return a `ModerationStatus` enum directly, rather than returning strings?